src/test/java/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,42 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringAddCalculatorTest {
+
+  @Test
+  public void split_쉼표구분자() throws Exception {
+
+    String[] result = "1,2".split(",");
+    assertThat(result).contains("1","2");
+  }
+
+  @Test
+  public void split_숫자하나() throws Exception {
+    String[] result = "1".split(",");
+    assertThat(result).contains("1");
+  }
+
+  @Test
+  public void clear_괄호() throws Exception {
+    String clear = "(1,2)".substring(1, 4);
+    assertThat(clear).isEqualTo("1,2");
+  }
+
+  @Test
+  public void find_특정위치문자() throws Exception {
+    String find = "(1,2)";
+    assertThat(find.charAt(1)).isEqualTo('1');
```

Comment:
```suggestion
    assertThat("(1,2").charAt(1)).isEqualTo('1');
```
로컬 변수 없이 위와 같이 구현해도 되지 않을까?